assembl/static2/js/app/components/common/imageUploader.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Translate } from 'react-redux-i18n';
+import { Button } from 'react-bootstrap';
+
+class ImageUploader extends React.Component {
+  constructor(props) {
+    super(props);
+    const { imgUrl } = this.props;
+    this.state = {
+      imgName: '',
+      imgUrl: imgUrl
+    };
+    this.handleChangePreview = this.handleChangePreview.bind(this);
+    this.handleUploadButtonClick = this.handleUploadButtonClick.bind(this);
+  }
+  handleUploadButtonClick() {
+    this.fileInput.click();
```

Comment:
here, we call the `click` event on the file input component (ref) when the button is clicked